library/blivet.py

```diff
@@ -579,7 +584,7 @@ def _create(self):
             try:
                 do_partitioning(self._blivet)
             except Exception as e:
-                raise BlivetAnsibleError("failed to allocate partitions for mdraid '%s': %s" % (raid_name, str(e)))
```

Comment:
would it make sense to replace `raid_name` by `self._volume["name"]` instead of removing it completely?

Reply:
Most of the information needed should be already included in e.

Reply:
I agree that the volume name could be useful here. The partitioning exception raised by blivet is not likely to indicate the name of the volume being configured.

Reply:
I see. Fixed.